src/sentry/issues/endpoints/organization_issue_metrics.py

```diff
@@ -0,0 +1,229 @@
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import EnvironmentMixin, region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.helpers.environments import get_environments
+from sentry.api.utils import get_date_range_from_params
+from sentry.issues.grouptype import GroupCategory
+from sentry.models.group import Group, GroupStatus
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+
+CATEGORY_MAP = {
+    "error": GroupCategory.ERROR,
+    "feedback": GroupCategory.FEEDBACK,
+}
+
+
+@region_silo_endpoint
+class OrganizationIssueMetricsEndpoint(OrganizationEndpoint, EnvironmentMixin):
+    owner = ApiOwner.REPLAY
+    publish_status = {"GET": ApiPublishStatus.PRIVATE}
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        """Stats bucketed by time."""
+        environments = [e.id for e in get_environments(request, organization)]
+        projects = self.get_projects(request, organization)
+        issue_category = request.GET.get("category", "error")
+        type_filter = (
+            ~Q(type=GroupCategory.FEEDBACK)
+            if issue_category == "error"
+            else Q(type=GroupCategory.FEEDBACK)
+        )
+        group_by = request.GET.get("group_by", "time")
+
+        # Start/end truncation and interval generation.
+        interval = timedelta(days=1)  # interval = request.GET.get("interval", "1d")
+        start, end = get_date_range_from_params(request.GET)
+        end = end.replace(hour=0, minute=0, second=0, microsecond=0) + interval
+        start = start.replace(hour=0, minute=0, second=0, microsecond=0)
+
+        if group_by == "time":
+            # Series queries.
+            new_series = query_new_issues(projects, environments, type_filter, start, end)
+            resolved_series = query_resolved_issues(projects, environments, type_filter, start, end)
+
+            # Filling and formatting.
+            series_response = empty_response(start, end, interval)
+            append_series(series_response, new_series)
+            append_series(series_response, resolved_series)
+        elif group_by == "release":
+            series_response = query_issues_by_release(
+                projects, environments, type_filter, start, end
+            )
+        else:
+            return Response("", status=404)
+
+        return Response(
+            {
+                "data": [[bucket, series] for bucket, series in series_response.items()],
+                "start": int(start.timestamp()),
+                "end": int(end.timestamp()),
+                # I have no idea what purpose this data serves on the front-end.
+                "isMetricsData": False,
+                "meta": {
+                    "fields": {"time": "date", "issues_count": "count"},
+                    "units": {"time": None, "issues_count": "int"},
```

Comment:
```suggestion
                    "units": {"time": None, "issues_count": None},
```
`issues_count` doesn't need a unit technically